frontend/src/lib/api/candidate.ts

```diff
@@ -189,15 +190,31 @@ export const getLikertQuestions = async (): Promise<Record<string, Question> | u
 
   const questionData: StrapiResponse<StrapiQuestionData[]> = await res.json();
 
-  const questions: Record<string, Question> = {};
+  const questions: Record<string, QuestionProps> = {};
 
   questionData.data.forEach((question) => {
-    questions[question.id] = {
+    const attr = question.attributes;
+    const settings = attr.questionType?.data.attributes.settings;
+    const props: QuestionProps = {
       id: `${question.id}`,
-      text: question.attributes.text,
-      editable:
-        question.attributes.category.data.attributes.election.data.attributes.canEditQuestions
+      text: translate(attr.text),
+      info: translate(attr.info),
+      shortName: translate(attr.shortName),
+      category: translate(question.attributes.category.data.attributes.name),
+      type: settings.type,
+      editable: attr.category.data.attributes.election.data.attributes.canEditQuestions
     };
+    if ('values' in settings)
+      props.values = settings.values.map(({key, label}) => ({
+        key,
+        label: translate(label)
+      }));
+    if ('min' in settings) props.min = settings.min;
+    if ('max' in settings) props.max = settings.max;
+    if ('dateType' in settings) props.dateType = settings.dateType;
+    if ('notLocalizable' in settings) props.notLocalizable = settings.notLocalizable;
```

Comment:
I think you could just directly define these in the props object:
```
const props = {
  min: settings.min,
  max: settings.max,
  ...
}
```

If the property is not defined, then it will also be undefined in the props object. I'm not 100% certain the functionality stays the same, but the current solution doesn't seem efficient.